pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -271,16 +107,16 @@ terminal:
 docker exec -it memgraph mgconsole
 ```
 
-You can also connect to Memgraph instance using the [Memgraph Platform desktop
-application](/data-visualization/user-manual), available for download from
-[Memgraph's Download Hub](https://memgraph.com/download). 
+You can also connect to Memgraph instance using the [Memgraph
+Lab](/data-visualization/user-manual), available for download from [Memgraph's
+Download Hub](https://memgraph.com/download). 
 
 To start using Memgraph in your application, use one of the [client
 libraries](/client-libraries) and follow their getting started guide.
 
 </Steps>
 
-### `memgraph/memgraph`
+### Install Memgraph Docker image
```

Comment:
Is it better to say Run Docker image, rather than Install?

Reply:
Fixed.